src/Libraries/Microsoft.Extensions.Http.Diagnostics/Logging/Internal/Log.cs

```diff
@@ -46,17 +46,114 @@ public static void OutgoingRequestError(ILogger logger, LogRecord record, Except
         OutgoingRequest(logger, LogLevel.Error, 2, nameof(OutgoingRequestError), record, exception);
     }
 
-    [LoggerMessage(LogLevel.Error, RequestReadErrorMessage)]
-    public static partial void RequestReadError(ILogger logger, Exception exception, HttpMethod httpMethod, string? httpHost, string? httpPath);
+    public static void RequestReadError(ILogger logger, Exception exception, HttpMethod method, string? host, string? path)
+    {
+        var state = LoggerMessageHelper.ThreadLocalState;
+
+        _ = state.ReserveTagSpace(4);
+        state.TagArray[3] = new(HttpClientLoggingTagNames.Method, method);
+        state.TagArray[2] = new(HttpClientLoggingTagNames.Host, host);
+        state.TagArray[1] = new(HttpClientLoggingTagNames.Path, path);
+        state.TagArray[0] = new(OriginalFormat, RequestReadErrorMessage);
```

Comment:
would be nice to add [exception.type](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/exceptions/exceptions-logs.md)  and other `exception` attributes

Reply:
This was implemented in another PR #4584.